std/math.d

```diff
@@ -4808,9 +4808,10 @@ unittest
     }
     ensureDefaults();
 
+    version(D_HardFloat)
     {
-        FloatingPointControl ctrl;
-        ctrl.rounding = FloatingPointControl.roundDown;
+        FloatingPointControl ctrlr;
+        ctrlr.rounding = FloatingPointControl.roundDown;
```

Comment:
OK, I'll see if it's already on bugzilla.  And you mean that `version` _will_ change the scope, right?  Because this `ctrl` was only defined inside the brackets, but adding the `version` made it defined outside.

Looks like something else broke now, I'll look at this more locally and update after a couple hours.